src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -149,24 +149,24 @@ public static SusiThought scrapeGithub(String profile) {
 			}
 		}
 		/* If Individual User */
-		if (html.getElementsByAttributeValue("class", "vcard-stat").size() != 0) {
+		if (html.getElementsByAttributeValueContaining("class", "user-profile-nav").size() != 0) {
 
-			String followersUrl = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(0).attr("href");
-			githubProfile.put("followers_url", "https://github.com" + followersUrl);
+			String starredUrl = "https://api.github.com/users/" + profile + "/starred";
+			githubProfile.put("starred_url", starredUrl);
 
-			String followers = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(0).tagName("strong").text();
-			githubProfile.put("followers", followers);
+			int starred = Integer.parseInt(html.getElementsByAttributeValue("class", "Counter").get(1).text());
+			githubProfile.put("starred", starred);
 
-			String starredUrl = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(1).attr("href");
-			githubProfile.put("starred_url", "https://github.com" + starredUrl);
+			String followersUrl = "https://api.github.com/users/" + profile + "/followers";
+			githubProfile.put("followers_url", followersUrl);
 
-			String starred = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(1).tagName("strong").text();
-			githubProfile.put("starred", starred);
+			int followers = Integer.parseInt(html.getElementsByAttributeValue("class", "Counter").get(2).text());
+			githubProfile.put("followers", followers);
 
-			String followingUrl = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(2).attr("href");
-			githubProfile.put("following_url", "https://github.com" + followingUrl);
+			String followingUrl = "https://api.github.com/users/" + profile + "/following";
+			githubProfile.put("following_url", followingUrl);
 
-			String following = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(2).tagName("strong").text();
+			int following = Integer.parseInt(html.getElementsByAttributeValue("class", "Counter").get(3).text());
 			githubProfile.put("following", following);
```

Comment:
Though I agree that all of this information would be useful, Why not create multiple requests from here for the API endpoints encountered and store that information too? One definitely needs to be careful about the request limit here for github's public APIs.

Do think about how as a developer you might interact with this API, What are the other content that you'd want to pass and check?
```
profile: - Users profile to scrape and store
terms: [ followers, following, starred, repos ]
```
If a user makes a request for profile, the current state of the scraper sounds good but if there are more additions to it in terms, maybe the user wants to know the `followers` of a specific user in `profile`, similarly with the other parameters.